content/ko/docs/concepts/policy/pod-security-policy.md

```diff
@@ -0,0 +1,618 @@
+| 컨테이너가 사용하는 sysctl 프로파일                          | [`forbiddenSysctls`,`allowedUnsafeSysctls`](#sysctl)                      |
+
+
+## 파드 보안 정책 활성화
+
+파드 보안 정책 제어는 선택 사항(하지만 권장함)인
+[어드미션 컨트롤러](/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy)로
+구현된다. [어드미션 컨트롤러 활성화](/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in)하면
+PodSecurityPolicy가 적용되지만
+정책을 승인하지 않고 활성화하면 클러스터에
+**파드가 생성되지 않는다.**
+
+파드 보안 정책 API(`policy/v1beta1/podsecuritypolicy`)는
+어드미션 컨트롤러와 독립적으로 활성화되므로 기존 클러스터의 경우
+어드미션 컨트롤러를 활성화하기 전에 정책을 추가하고 권한을
+부여하는 것이 좋다.
+
+## 정책 승인
+
+PodSecurityPolicy 리소스가 생성되면 아무 것도 수행하지 않는다. 이를 사용하려면
+요청 사용자 또는 대상 파드의
+[서비스 어카운트](/docs/tasks/configure-pod-container/configure-service-account/)는
+정책에서 `use` 동사를 허용하여 정책을 사용할 권한이 있어야 한다.
+
+대부분의 쿠버네티스 파드는 사용자가 직접 만들지 않는다. 대신 일반적으로
+컨트롤러 관리자를 통해
+[디플로이먼트](/ko/docs/concepts/workloads/controllers/deployment/),
+[레플리카셋](/ko/docs/concepts/workloads/controllers/replicaset/), 또는 기타
+템플릿 컨트롤러의 일부로 간접적으로 생성된다. 컨트롤러에 정책에 대한 접근 권한을 부여하면
+해당 컨트롤러에 의해 생성된 *모든* 파드에 대한 접근 권한이 부여되므로 정책을 승인하는
+기본 방법은 파드의 서비스 어카운트에 대한 접근 권한을 부여하는 것이다([예](#다른-파드를-실행) 참고).
+
+### RBAC을 통한 방법
+
+[RBAC](/docs/reference/access-authn-authz/rbac/)은 표준 쿠버네티스 권한 부여 모드이며
+정책 사용 권한을 부여하는 데 쉽게 사용할 수 있다.
+
+먼저 `Role` 또는 `ClusterRole`은 원하는 정책을 `use` 하려면 접근 권한을 부여해야 한다.
+접근 권한을 부여하는 규칙은 다음과 같다.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: <role name>
+rules:
+- apiGroups: ['policy']
+  resources: ['podsecuritypolicies']
+  verbs:     ['use']
+  resourceNames:
+  - <list of policies to authorize>
+```
+
+그런 다음 `(Cluster)Role`이 승인된 사용자에게 바인딩된다.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: <binding name>
+roleRef:
+  kind: ClusterRole
+  name: <role name>
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+# Authorize specific service accounts:
+- kind: ServiceAccount
+  name: <authorized service account name>
+  namespace: <authorized pod namespace>
+# Authorize specific users (not recommended):
+- kind: User
+  apiGroup: rbac.authorization.k8s.io
+  name: <authorized user name>
+```
+
+`RoleBinding`(`ClusterRoleBinding` 아님)을 사용하는 경우 바인딩과 동일한 네임스페이스에서
+실행되는 파드에 대해서만 사용 권한을 부여한다. 네임스페이스에서 실행되는 모든 파드에 접근 권한을
+부여하기 위해 시스템 그룹과 쌍을 이룰 수 있다.
+```yaml
+# Authorize all service accounts in a namespace:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: system:serviceaccounts
+# Or equivalently, all authenticated users in a namespace:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: system:authenticated
+```
+
+RBAC 바인딩에 대한 자세한 예는 [역할 바인딩 예제](/docs/reference/access-authn-authz/rbac#role-binding-examples)를 참고하길 바란다. PodSecurityPolicy 인증에 대한 전체 예제는
+[아래](#예제)를 참고하길 바란다.
+
+
+### 문제 해결
+
+- [컨트롤러 관리자](/docs/admin/kube-controller-manager/)는
+[보안 API 포트](/docs/reference/access-authn-authz/controlling-access/)에 대해 실행해야 하며
+슈퍼유저 권한이 없어야 한다. 그렇지 않으면 요청이 인증 및 권한 부여 모듈을 무시하고
```

Comment:
> origin
> Otherwise requests would bypass authentication and authorization modules, all PodSecurityPolicy objects would be	

```suggestion
슈퍼유저 권한이 없어야 한다. 그렇지 않으면 요청이 인증 및 권한 부여 모듈을 우회하고,
```

- [  ] `bypass`를 `우회하다`로 번역하면 더 자연스러울 것 같습니다.

Reply:
네, 훨씬 자연스러운것 같습니다.